kevm-pyk/src/kevm_pyk/kproj/evm-semantics/buf.md

```diff
@@ -41,6 +41,7 @@ module BUF
 
     rule 0    <Int #powByteLen(SIZE) => true requires 0 <=Int SIZE [simplification]
     rule SIZE <Int #powByteLen(SIZE) => true requires 0 <=Int SIZE [simplification]
+    rule #write(WM, IDX, VAL) => WM [ IDX := #buf(1, VAL) ] [symbolic(WM), simplification]
```

Comment:
This could potentially not be needed, but should apply at all times really, independently of whether the memory/index/value are concrete or symbolic.